md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -222,6 +222,31 @@ If you did not set the default Look & Feel before migration and you cannot log i
 
 The migration is now complete. If you were using a custom Look & Feel before migration, test it on the new version before applying it to your migrated platform.
 
+## Migration of processes with v6 forms
+
+Until the version 7.0.0, Bonita used forms based on the GWT technology, for the instantiation of processes, tasks and process overview.
+The runtime support for those forms was removed in 7.8.0.
+
+It means, that if one or more processes on the migrated server uses v6 forms, the migration to a version above 7.7.4 cannot be performed directly. The following lines explain how to migrate to a version 7.8.0 +
+
+Specifically if you are migrating from a 6.x version:
+* Migrate to the 7.0.0 using the migration tool 1.x.
+* Migrate to the 7.7.4 using the migration tool 2.x.
+* Redesign all your forms in the Studio using the UI designer. See [here](migrate-a-form-from-6-x.md) for more info.
+* Upload the new version of all your processes using the new forms.
+* Disable the version of your processes using v6 forms. Make sure they have no more running instances.
+* Perform the migration to the desired version.
+
+If you are migrating from a 7.x version :
+* Redesign all your forms in the Studio using the UI designer. See [here](migrate-a-form-from-6-x.md) for more info.
```

Comment:
What is the difference between 6.x and 7.x with 6.x forms?

Reply:
None, aside of the fact that you can't redesign your 6.x forms in a 6.5.4 ;).

Reply:
Alright. The sentences look the same, so I was wondering why not making them only one sentence. Not a big deal anyway.